controller/log_test.go

```diff
@@ -12,10 +12,10 @@ import (
 	"sync"
 	"time"
 
+	. "github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-check"
 	ct "github.com/flynn/flynn/controller/types"
 	logaggc "github.com/flynn/flynn/logaggregator/client"
-
-	. "github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-check"
+	. "github.com/flynn/flynn/pkg/typeconv"
```

Comment:
I don't think we should add more dot-imports other than gocheck. https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Import_Dot